packages/gdc/src/GDCFeatureDrawerWidget/GDCFeatureDrawerWidget.js

```diff
@@ -0,0 +1,750 @@
+import Divider from '@material-ui/core/Divider'
+import Paper from '@material-ui/core/Paper'
+import { makeStyles } from '@material-ui/core/styles'
+import Table from '@material-ui/core/Table'
+import TableBody from '@material-ui/core/TableBody'
+import TableCell from '@material-ui/core/TableCell'
+import TableHead from '@material-ui/core/TableHead'
+import TableRow from '@material-ui/core/TableRow'
+import Chip from '@material-ui/core/Chip'
+import Tooltip from '@material-ui/core/Tooltip'
+import Link from '@material-ui/core/Link'
+import Icon from '@material-ui/core/Icon'
+import { observer, PropTypes as MobxPropTypes } from 'mobx-react'
+import PropTypes from 'prop-types'
+import React, { useState, useEffect } from 'react'
+import BaseFeatureDetail, {
+  BaseCard,
+} from '@gmod/jbrowse-core/BaseFeatureDrawerWidget/BaseFeatureDetail'
+
+const useStyles = makeStyles(() => ({
+  table: {
+    padding: 0,
+  },
+  title: {
+    fontSize: '1em',
+  },
+  link: {
+    color: 'rgb(0, 0, 238)',
+    'text-decoration': 'underline',
```

Comment:
Should probably use `underline="always"` on the `Link` component instead of text-decoration. (Also FYI, this will understand camelCase-ified CSS props, too, so if you did use it you could use `textDecoration: 'underline'`)

Reply:
Didn't realize I could do camel case CSS props, good to know.